README.md

```diff
@@ -80,6 +80,8 @@ grails:
                 id: 'id'
                 email: 'emailAddress'
                 username: 'login'
+                data:
+                    - 'authorities' # Additional properties the principal
```

Comment:
I was a bit confused at the beginning so maybe better wording would help:

> Additional properties to be retrieved from user details object and passed as extra properties to Sentry user interface

or something like that?